test/gtest/ucp/test_ucp_am.cc

```diff
@@ -811,12 +824,13 @@ class test_ucp_am_nbx_closed_ep : public test_ucp_am_nbx {
         return ep_params;
     }
 
-    void test_recv_on_closed_ep(size_t size, unsigned flags = 0,
-                                bool poke_rx_progress = false,
+    void test_recv_on_closed_ep(size_t size, bool poke_rx_progress = false,
                                 bool rx_expected = false)
     {
+        unsigned flags = get_send_flag();
+
         skip_loopback();
-        test_am_send_recv(0, max_am_hdr()); // warmup wireup
+        test_am_send_recv(0, max_am_hdr(), flags); // warmup wireup
```

Comment:
do we really need to pass flags?
given https://github.com/openucx/ucx/blob/07152a5fbc229be51e3564d7450d6957073314f0/test/gtest/ucp/test_ucp_am.cc#L456

Reply:
yes, we don't need
removed